setup.cfg

```diff
@@ -1,7 +1,9 @@
 [flake8]
+exclude = .git,.env
 # This section configures `flake8`, the python linting utility.
 # See also https://flake8.pycqa.org/en/latest/user/configuration.html
-ignore = D100,D101,D102,D103,D105,D200,D205,D210,D400,D401,D403,E127,E201,E202,E203,E221,E222,E241,E265,E271,E272,E301,E501,N802,N803,N805,N806,W503
+ignore = D100,D101,D102,D103,D105,D200,D205,D210,D400,D401,D403,E203,E501,N802,N803,N804,N805,N806,W503
```

Comment:
Slimming down the list 😌

Reply:
Yep, we can work on migrating these to all of the repos later